lib/parzival/companies.ex

```diff
@@ -805,12 +805,9 @@ defmodule Parzival.Companies do
       {:error, %Ecto.Changeset{}}
 
   """
-  def create_connection(%Company{} = company, %User{} = user) do
+  def create_connection(attrs \\ %{}) do
```

Comment:
Why have you refactored this function? It was doing a very good job, pattern matching the arguments.
The approach to follow in this cases should be to fix the test, not to _fix_ the function 😅